SpatialGDK/Source/SpatialGDKFunctionalTests/SpatialGDK/EventTracingTests/EventTracingTest.cpp

```diff
@@ -128,29 +150,31 @@ void AEventTracingTest::GatherData()
 		return A.CreationTime > B.CreationTime;
 	});
 
-	bool bFoundClient = false;
-	bool bFoundWorker = false;
+	FPlatformProcess::Sleep(1); // Worker bug means file may not be flushed by the OS (WRK-2396)
```

Comment:
Alternatively we could just close the stream, I think I prefer that over a sleep

Reply:
In the future we can just use the flush API worker provide